index.js

```diff
@@ -157,10 +157,12 @@ function isKnownIncompatiblePluginFromState(state) {
 }
 
 function attributeNamesFromState(state) {
-  if (state.opts[nativeOptionName] === true) {
```

Comment:
`nativeOptionName` becomes unused. We may want to delete the variable or use the previous pattern.

Reply:
Oh, right. I'll delete it.